trial-config-service/src/test/resources/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "trial": [
```

Comment:
trial or trials?

Reply:
not sure - I think we should leave as trial until clarification

Reply:
file has been removed as a new testing approach has been agreed.